app/controllers/publishers/site_banners_controller.rb

```diff
@@ -23,24 +113,27 @@ def create
     head :ok
   end
 
-  def fetch
-    if current_publisher.site_banner.present?
-      data = current_publisher.site_banner.read_only_react_property
-      data[:backgroundImage] = data[:backgroundUrl]
-      data[:logoImage] = data[:logoUrl]
-      render(json: data.to_json)
-    else
-      render(json: nil.to_json)
-    end
-  end
+  def create
+   site_banner = current_publisher.site_banners || SiteBanner.new
+   donation_amounts = JSON.parse(params[:donation_amounts])
+   site_banner.update(
+     publisher_id: current_publisher.id,
+     title: params[:title],
+     donation_amounts: donation_amounts,
+     default_donation: donation_amounts.second,
+     social_links: params[:social_links].present? ? JSON.parse(params[:social_links]) : {},
+     description: params[:description]
+   )
+   head :ok
+ end
```

Comment:
`site_banner = current_publisher.site_banners || SiteBanner.new`

What happens if this evaluates to `current_publisher.site_banners`?

Reply:
Done ✔️ (Removed this function)